src/Sylius/Bundle/CoreBundle/Controller/OrderController.php

```diff
@@ -111,4 +124,159 @@ public function historyAction(Request $request)
 
         return $this->viewHandler->handle($configuration, $view);
     }
+
+    /**
+     * @param Request $request
+     * @param int $orderId
+     *
+     * @return Response
+     */
+    public function payAction(Request $request, $orderId)
```

Comment:
@Arminek I'd say they are dealing with Order as "main" entity and I think we should keep them here, at least for now.